Sources/CSharpStandAlone/Generator.swift

```diff
@@ -0,0 +1,121 @@
+import Lexicon
+import UniformTypeIdentifiers
+
+public enum Generator: CodeGenerator {
+    
+    // TODO: prefixes?
+    
+    public static let utType = UTType(filenameExtension: "cs", conformingTo: .sourceCode)!
+    
+    public static func generate(_ json: Lexicon.Graph.JSON) throws -> Data {
+        return Data(json.cSharp().utf8)
+    }
+}
+
+private extension Lexicon.Graph.JSON {
+    
+    func cSharp() -> String {
+        return """
+public interface I_TypeLocalised
+{
+    string Localised { get; set; }
```

Comment:
In the swift version this is static, since you can only reliably derive the localisation from that context